astroplan/core.py

```diff
@@ -2,6 +2,21 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
+from astropy.coordinates import (EarthLocation, Latitude, Longitude, SkyCoord,
+                                 AltAz)
+import astropy.units as u
+from astropy.units.quantity import Quantity
+
+import pytz
+
+################################################################################
+# Temporary solution to IERS tables problems
```

Comment:
Add a TODO marker, or even better yet, make a Github issue that this should be changed / removed once X in Astropy happens. @bmorris3 – You probably know what we are waiting for and what should be done, but who knows, maybe it'll take a year to be resolved, and then the next person working on astroplan doesn't. (And it can be nice for yourselt too, to have a list of issues that can be addressed one by one instead of having to remember things.)